### ISSUE TITLE
Not working on centos7

### ISSUE THREAD

**Issue by crstin** [mattmc3/antidote#89]
Both `zsh 5.0.2 (x86_64-redhat-linux-gnu)` (default) and `zsh 5.9 (x86_64-pc-linux-gnu)` give

```sh
(anon):14: unknown file attribute
```

**Comment by mattmc3** [comment]
> zsh 5.0.2 (x86_64-redhat-linux-gnu)

Zsh versions prior to 5.4.2 are not supported, and won't be. 5.4.2 shipped with [Ubuntu 18.04 LTS](https://packages.ubuntu.com/bionic/zsh), and is verified to work via [GitHub workflows](https://github.com/mattmc3/antidote/blob/main/.github/workflows/test.yml) in this project.

> zsh 5.9 (x86_64-pc-linux-gnu)

This one is much more surprising. Can you give any more detail? Can you verify that it's really failing on 5.9 (`/usr/bin/env zsh -c 'echo $ZSH_VERSION'; /bin/zsh -c 'echo $ZSH_VERSION'`).

**Comment by crstin** [comment]
Sure

```sh
/usr/bin/env zsh -c 'echo $ZSH_VERSION'; /bin/zsh -c 'echo $ZSH_VERSION'
5.9
5.0.2

which -a zsh
/usr/local/bin/zsh
/usr/bin/zsh

zsh --version
zsh 5.9 (x86_64-pc-linux-gnu)

source antidote.zsh
(anon):14: unknown file attribute
```
